examples/forms.amp.html

```diff
@@ -238,15 +238,12 @@
     </fieldset>
 
     <div submit-success>
-        <template type="amp-mustache">
-            Success! Thanks {{name}} for subscribing! Please make sure to check your email {{email}}
-        to confirm!
-        </template>
+      Success! Thanks for subscribing! Please make sure to check your email
```

Comment:
Don't have a strong opinion. As long as there are other usages in this example file that use `template`, we should be fine either way.